proposals/coredns.adoc

```diff
@@ -0,0 +1,82 @@
+== CoreDNS Proposal
+
+*Name of project:* CoreDNS
+
+*Description*
+
+CoreDNS is a fast, flexible DNS server that is designed to be performant with a flexible implementation. This flexibility allows CoreDNS to be easily extended to support various data sources and to implement rich DNS service behaviors. For example, response caching, query rewrite, load-balancing, zone transfer and signing.
+CoreDNS is the successor of SkyDNS.
+
+*Sponsor / Advisor from TOC*: Jonathan Boulle
+
+*Unique Identifier*: coredns
+
+*License*: Apache License v2.0
+
+*Source control repositories*: https://github.com/miekg/coredns
+
+*Initial Committers*:
+
+* Michael Richmond github: mrichmon
+* github: splack
+* Felix Cantournet github: fcantournet
+* github: leelynne
+* Matt Layher github: mdlayher
+* Vasily Vailyev github: pixelbender
+
+*Infrastructure requirements (CI / CNCF Cluster)*: N/A
+
+*Issue tracker*: https://github.com/miekg/coredns
+
+*Website*: https://coredns.io
+
+*Release methodology and mechanics*: github master
+
+*Social media accounts*: Twitter: @corednsio
+
+*Existing sponsorship*: Infoblox contributing developer time to implement CoreDNS->Kubernetes integration component.
+
+*External Dependencies*
```

Comment:
Could you talk a bit about the relationship with Caddy? from our discussion on the call the other day it sounded like Caddy will become a dependency once CoreDNS is ported over (which looks like it's happening [soon](https://github.com/miekg/coredns/pull/220)?)